fakeData.py

```diff
@@ -0,0 +1,135 @@
+import numpy as np
+import random
+from faker import Faker
+import pandas as pd
+
+"""
+
+Copmpile a fake dataset resembling the dataset Michael collected on his trip to Guatemala in.
+
+The fake dataset uses relative proportions of the tree types in the original dataset as 
+probabilities for selection of the tree types of this dataset. So, although it is not the original data--
+and the names are fake!--it resembles the patterns of the co-op.
+
+"""
+
+fake = Faker('es_MX') # faker does not have options for any American Spanish varieties besides Mexico,
```

Comment:
add comment above this line. just a stylistic thing but I tend to avoid in-line comments if they exceed like, ten-fifteen characters.